api/exclusions/exclusionService.ts

```diff
@@ -31,8 +31,15 @@ export async function confirmUserExclusion(req: EnhancedRequest, res: Response,
       userType: 'Judicial',
     },
   ];
-  if (['400, 401, 402, 403, 500, 503'].indexOf(req.body.exclusionDescription)) {
-    return res.status(req.body.exclusionDescription).send('error: {status: req.body.exclusionDescription}"');
+  if (req.body.exclusionDescription && ['400, 401, 402, 403, 500, 503'].indexOf(req.body.exclusionDescription)) {
+    return res.status(req.body.exclusionDescription).send(`error: {status: ${req.body.exclusionDescription}}`);
   }
-  return res.send(exclusion).status(200);
+  return res.status(200).send(exclusion);
+}
+
+export async function deleteUserExclusion(req: EnhancedRequest, res: Response, next: NextFunction) {
+  if (req.body.exclusionDescription && ['400, 401, 402, 403, 500, 503'].indexOf(req.body.exclusionDescription)) {
```

Comment:
for the delete journey, as we couldn't control what is inside the req.body.exclusionDescription to trigger the unhappy path, is this necessary to use the same mock logic like add exclusion.

Reply:
no, I can remove it